addOns/graphql/src/main/java/org/zaproxy/addon/graphql/automation/GraphQlJob.java

```diff
@@ -199,19 +202,27 @@ public Parameters getParameters() {
         return parameters;
     }
 
+    @Getter
     public static class Data extends JobData {
-        private Parameters parameters;
+        private final Parameters parameters;
```

Comment:
Wdyt about moving parameters to `JobData`? That will ensure that we are consistent with the field name across jobs. Each job already provides its parameters (by overriding `getParameters`).

Reply:
We _have_ to support backwatds compatibility.
If thats maintained then we should go with whatever structure we think makes the most sense 😁 
Oh, and without changing things to often 😉

Reply:
How is it going to be declared in the `JobData`? It's not the same type in all jobs.

Reply:
Right, I was thinking of using `AutomationData`...